support-frontend/assets/pages/supporter-plus-landing/twoStepPages/checkoutScaffold.tsx

```diff
@@ -35,17 +38,20 @@ import {
 	UnitedStates,
 } from 'helpers/internationalisation/countryGroup';
 import { useContributionsSelector } from 'helpers/redux/storeHooks';
+import { LandingPageHeading } from 'pages/digital-subscriber-checkout/components/landingPageHeading';
 import { CheckoutDivider } from '../components/checkoutDivider';
 import { GuardianTsAndCs } from '../components/guardianTsAndCs';
-import { LandingPageHeading } from '../components/landingPageHeading';
 import { PatronsMessage } from '../components/patronsMessage';
 
-const checkoutContainer = (isPaymentPage?: boolean) => css`
+const checkoutContainer = (
+	isPaymentPage?: boolean,
+	showUsEoy2023Content?: boolean,
+) => css`
 	position: relative;
 	color: ${palette.neutral[7]};
 	${textSans.medium()};
 
-	padding-top: ${space[isPaymentPage ? 2 : 6]}px;
+	padding-top: ${space[isPaymentPage ? 2 : !showUsEoy2023Content ? 3 : 6]}px;
```

Comment:
this could be:

```
) => {
  let paddingTop = 6;
  if (isPaymentPage) {
    paddingTop = 2;
  } else if (!showUsEoy2023Content) {
    paddingTop = 3;
  }
  return css`
    position: relative;
    color: ${palette.neutral[7]};
    ${textSans.medium()};
    padding-top: ${space[paddingTop]}px;

Reply:
might be a bit more readable than the double ternary

Reply:
This gives me -> 'Element implicitly has an 'any' type because expression of type 'number' can't be used to index type '{ readonly 1: 4; readonly 2: 8; readonly 3: 12; readonly 4: 16; readonly 5: 20; readonly 6: 24; readonly 9: 36; readonly 12: 48; readonly 24: 96; }'

Reply:
paddingTop cant be a number need to force to be 2, 3 or 6 only

Reply:
Ok, seems a lot of code to replace a line but definately more readable, had to limit the type of `paddingTop` to 2,3 or 6 however ( `type SpaceRange = 2 | 3 | 6;` added )